x-pack/test/functional/apps/maps/group1/sample_data.js

```diff
@@ -138,7 +138,7 @@ export default function ({ getPageObjects, getService, updateBaselines }) {
           'web_logs_map',
           updateBaselines
         );
-        expect(percentDifference).to.be.lessThan(0.02);
```

Comment:
Can you add some context for increasing this threshold. Maybe some screen shots from each or a diff screen shot. A 50% increase is pretty big

Reply:
To be honest, I've struggled with Kibana's visual snapshot tests in the past. When I've tried to update the baselines locally previously, the new images appear to be using a totally different set of DPI/dimensions than CI and it ends up failing completely.

I'd appreciate either being able to pair with someone on your team for that, or alternatively having your team update baselines in a follow-up PR after this one. Please let me know what you prefer!

Reply:
> or alternatively having your team update baselines in a follow-up PR after this one. Please let me know what you prefer!

We can just make a PR to update the snapshots afterwards. That will be easier for you

Reply:
Thanks a million @nreese!